ShortenLink/Constants/Constants.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Constants.swift
+//  ShortenLink
+//
+//  Created by Su T. Nguyen on 21/11/2021.
+//
+
+import Foundation
+
+enum Constants {
+
+    enum API { }
+    enum Google { }
+}
+
+extension Constants.API {
+
+    static let baseURL: String = "https://nimble-link.herokuapp.com/"
+}
+
+extension Constants.Google {
+
+    static let clientId: String = "659972231175-tsph0d8ucfnntotncki6nle4jqks1t6a.apps.googleusercontent.com"
```

Comment:
Is this safe to store in Github? Should we store it locally only...? We could use .env file but it will take more time to develop.

Reply:
yup, I totally agree with you, it's better practical, we will leave that task for the next version 🤣